src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -1171,9 +1089,8 @@ namespace HVACUnitaryBypassVAV {
                 ShowContinueError("In " + CurrentModuleObject + " \"" + CBVAV(CBVAVNum).Name + "\".");
             }
 
-            if (CBVAV(CBVAVNum).DXCoolCoilType_Num > 0) {
-                ControlNodeNum = GetOnlySingleNode(
-                    Alphas(7), ErrorsFound, CurrentModuleObject, Alphas(1), NodeType_Air, NodeConnectionType_Sensor, 1, ObjectIsParent);
+            if (NumNumbers > 8) {
+                CBVAV(CBVAVNum).minModeChangeTime = Numbers(9);
```

Comment:
new input for minimum operating time in cooling or heating. If not entered, min time = 0, if field is present and blank default is read in as 0.25 hr, else read user input.